create_redshift_table.R

```diff
@@ -0,0 +1,38 @@
+library(buffer)
+
+# create redshift table for events
+create_table <- function(connection) {
+  
+  print("Creating Redshift table")
+  
+  # connect to redshift
+  con <- redshift_connect()
+  
+  # define create statement
+  table_recipe <- "
+    create table manual_mrr_amounts(
+      customer_id varchar(256) encode lzo,
+      buffer_user_id varchar(256) encode lzo,
```

Comment:
Curious about this field, does it only apply to Publish users or could it also be set for Reply users?

Reply:
Good question. This is only Publish users. Should we change the column name to `publish_user_id`?

Reply:
Good one!  Yeah, I think `publish_user_id` feels better to me, as it would be a bit more clear as we move more onto multi-prod data models 😄

Reply:
Ok cool! I renamed that column and updated the name in the Rscript too here: https://github.com/bufferapp/manual-payments/pull/8/commits/3fbc3dc98b000827ce971fde6840dbebd12509f5